README.adoc

```diff
@@ -167,10 +171,10 @@ only when the annotated method is called and decrease when it returns
 | `description` | Same as above
 |===
 
-The `@Gauge` annotation on the `getTotal()` method either tracks the inventory size or the number of systems that are stored
-in the inventory. A system is stored in the inventory when you visit the
+The `@Gauge` annotation on the `getTotal()` method tracks the number of systems that are stored in 
+the inventory. A system is stored in the inventory when you visit the 
 `http://localhost:9080/inventory/systems/localhost` URL to retrieve the information for a specific host.
-Below are the explanation of the fields:
+This is the additional metadata field: 
```

Comment:
The following table shows the additional metadata field:

Reply:
I like "Here is the additional metadata field:"

Reply:
Fixed

Reply:
=> "See the additional metadata field:"

Reply:
Fixed